source/crud/crud.rst

```diff
@@ -336,165 +338,209 @@ Read
     /**
      * Get partial results from a mongos if some shards are down (instead of throwing an error).
      *
-     * The default in servers >= 3.2 is no value: no "allowPartialResults" option is sent with
-     * the "find" command.
-     *
-     * In servers < 3.2, the Partial OP_QUERY flag defaults to false.
+     * This option is sent only if the caller explicitly provides a value. The default is to not send a value.
+     * For servers < 3.2, the Partial wire protocol flag defaults to false.
```

Comment:
No other text for this option refers to "Partial wire protocol flag" and this is the only indication that it relates to the wire protocol. Would it be better to omit this line or say that the effective behavior on an older server is false without referring to the wire protocol?